lms/djangoapps/teams/views.py

```diff
@@ -94,7 +95,9 @@ def get(self, request, course_id):
                 'topics_detail', kwargs={'topic_id': 'topic_id', 'course_id': str(course_id)}, request=request
             ),
             "topics_url": reverse('topics_list', request=request),
-            "teams_url": reverse('teams_list', request=request)
+            "teams_url": reverse('teams_list', request=request),
+            "username": user.username,
+            "privileged": user.is_staff or CourseStaffRole(course_key).has_user(user)
```

Comment:
I don't think you are testing for the right things here. We only want privileged to be Discussion Admin, Moderator, or TA. And we don't want people with generic is_staff access to be marked as privileged.